src/lib/components/ui/ScrollView/ScrollView.jsx

```diff
@@ -75,6 +77,22 @@ export const ScrollView = (props) => {
     debounce,
   );
 
+  const autoScrollChildrenHash = autoScroll && children && React.Children
+    .map(children, (child) => child.key)
+    .reduce((reducedHash, childHash) => reducedHash + childHash, '');
```

Comment:
I think the names `reducedHash` and `childHash`  are misleading. They are no hashes but the `key` attributes.